lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -370,7 +375,14 @@ void LayerTreeItem::saveAnonymousLayer()
 
             // now replace the layer in the parent
             if (isRootLayer()) {
-                sessionState->rootLayerPathChanged(fileName);
+                const bool makePathRelative
+                    = MGlobal::optionVarExists(MAKE_PATH_RELATIVE_TO_SCENE_FILE)
+                    && MGlobal::optionVarIntValue(MAKE_PATH_RELATIVE_TO_SCENE_FILE);
+
+                sessionState->rootLayerPathChanged(
+                    makePathRelative
+                        ? UsdMayaUtilFileSystem::getPathRelativeToMayaSceneFile(fileName)
+                        : fileName);
```

Comment:
Same comment as before, we need to handle the cases where we can't make the path relative even when requested.